src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +19,12 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        return movieDao.get(id).orElseThrow(() ->
+                new RuntimeException("Movie not found by id: " + id));
```

Comment:
Consider using `DataProcessingException` instead of `RuntimeException` for consistency with other service implementations. This will provide a more specific context for the error when a movie is not found.